src/main/play-doc/operation/CLI.md

```diff
@@ -90,52 +90,6 @@ shazar ./visualizer-poll-interval.sh
 
 The configuration is now ready to be loaded along with a bundle. Configuration is always provided as the second param to the `conduct load` command.
 
-## Deploying bundles
```

Comment:
I was thinking whether *Deploying bundles* section should stay here or separate section myself.

What made me pull it out into a separate section is I want to consolidate all deployment related topics into one deployment section. Also the deployment section being a top level link called *Deploying Bundles* under Operations is a plus, I think.

I did read to see how the flow feels like reading from CLI to the next section. I feel it's less jarring this way compared to having *Deploying bundles* in a separate section from shorthand and orchestration.

I initially started having everything under CLI section, and it does feel that it becomes too long read.

Perhaps maybe we can split the section:
- *CLI* section explains how to install the CLI tool and what it is for
- *Deploying Bundles* section explains how to deploy
- *Customising Deployment Configuration* section explains how to override config (i.e. shazar command)
- *Listing Bundle Endpoints* section explains about `conduct services`